doc/src/task-job-submission.rst

```diff
@@ -278,6 +278,40 @@ directive explicitly if ``execution time limit`` is specified.
 Otherwise, the execution time limit known by the suite may be out of sync with
 what is submitted to the batch system.
 
+Cylc supports heterogeneous Slurm jobs via special numbered directive prefixes
+that distinguish repeated directives from one another:
+
+.. code-block:: cylc
+
+   [runtime]
+       # run two heterogenous job components:
+       script = srun sleep 10 : sleep 30
+       [[my_task]]
+           [[[job]]]
+               batch system = slurm
+               execution time limit = PT1H
+           [[[directives]]]
+               --account = QXZ5W2
+               hetjob_0_--mem = 1G  # first prefix must be "0"
```

Comment:
Possible user unclarity:
If I want more than one directive for `hetjob_0_` do I add a newline, or append to this line? - that is:
```
hetjob_0_--mem = 1G
hetjob_0_--gres = 6Apples
```
or 
```
hetjob_0_--mem = 1G --gres=5Peaches
```
Could you add a second directive for one or other hetjob to address this?

Reply:
Good point @wxtim - it's the former. I'll update the documentation to add a second directive then merge this (as the change will not affect the code at all).